io.github.qier222.YesPlayMusic.yaml

```diff
@@ -0,0 +1,71 @@
+app-id: io.github.qier222.YesPlayMusic
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+separate-locales: false
+rename-icon: yesplaymusic
+rename-desktop-file: yesplaymusic.desktop
+command: yesplaymusic
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+  - --filesystem=xdg-config/fontconfig:ro
```

Comment:
why do you need this

Reply:
--socket=pulseaudio  for playing music
--device=dri    for hardware acceleration when playing music video
--share=network  network is required to play music from [netease cloud music](https://music.163.com/)
--filesystem=xdg-config/fontconfig:ro  follow user's custom font configuration

Reply:
deleted xdg-config/fontconfig:ro